features/user_sign_in.feature

```diff
@@ -0,0 +1,32 @@
+Feature: As a user
+  In order to use the application
+  I should be able to sign in to my account
+
+  Background:
+    Given the following user exists
+      | id      | email               | encrypted_password |
+      | 1       | example@example.com | password           |
+    And I navigate to the index page
```

Comment:
Why do you feel you need to have id in the table

Reply:
I thought I needed it I will remove it :)